python/utils/color.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from sgtk.platform.qt import QtGui
+
+
+def color_mix(c1, c1_strength, c2, c2_strength):
```

Comment:
by putting this in the top level of shotgun utils i think we are making it a "component" that people can import into their apps - so we should make sure that when this is imported by for example the hierarchy model, it also uses import_module() rather than a .. style import (for the sphinx build).

For the shotgun model, i wasn't sure if my utility methods would be useful outside of the shotgun model so i added it as a `util.py` file here : https://github.com/shotgunsoftware/tk-framework-shotgunutils/blob/master/python/shotgun_model/util.py

I think in this case, we should probably do the same and keep it internal within the hierarchy model for the time being. What do you think?